x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -1344,16 +1344,10 @@ private <T> void findApiKeysForUserRealmApiKeyIdAndNameCombination(
     /**
      * Performs the actual invalidation of a collection of api keys
      *
-     * @param apiKeyIds       the api keys to invalidate
-     * @param listener        the listener to notify upon completion
-     * @param previousResult  if this not the initial attempt for invalidation, it contains the result of invalidating
```

Comment:
`previousResult` was always `null` so simplifying the code here

Reply:
I think this pattern was copied from TokenService. But it is not useful here.